gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationFragment.java

```diff
@@ -197,6 +203,12 @@ public void onShowDialog(Field field) {
     }
   }
 
+  public void onShowPhotoSelectorDialog(Field field) {
```

Comment:
Can we somehow use butterknife binding  `butterknife.BindView` here instead instead of calls to show, etc.?

Reply:
I'm okay with either options. Just wanted to be consistent with other places. Should I replace this with `Butterknife` and remove `DataBinding`?

Reply:
Ah, all good--I think my initial comment is irrelevant here since this is dialog frag. 

This should be fine.